### DISCUSSION TITLE
How can i use ‘union all’

### DISCUSSION THREAD

**Discussion by imejiasoft** [prisma/prisma#7540] [Q&A]
Hi, i need to get a list from somes tables without relations and after that pagination but i dont want everything

**Comment by ryands17** [comment]
@imejiasoft 👋 

Currently this is only possible via a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) via `$queryRaw`. I would suggest creating a feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
